magento/magento2
# Update Magento logo & copyright to add Adobe

Add new Magento logo with Claim: "an Adobe company"
Add new Copyright with claim: "an Adobe company"



### Description


### Fixed Issues (if relevant)


### Manual testing scenarios

1. Go to Admin
2. Check Logo
3. Check CopyRight
4. Change Theme for Blank
5. Check Logo

### Contribution checklist
